src/datastore/postgres-store.ts

```diff
@@ -1217,6 +1219,9 @@ export class PgDataStore
             await this.updateNamespaces(client, entry.tx, namespace);
           }
         }
+        if (newNftEvents) {
+          await client.query(`REFRESH MATERIALIZED VIEW nft_custody`);
```

Comment:
@rafaelcr shouldn't this refresh for macroblocks updates also? https://github.com/hirosystems/stacks-blockchain-api/blob/b9a8c4a60044e2713c882df5326acca42dfd5bc2/src/datastore/postgres-store.ts#L1409

Reply:
Right now the materialized view only keeps custody status at the canonical chain tip. However, this is a good question that I think @zone117x can answer: if a user requests NFT events with `includeUnanchored` set to true, I would need to ignore the materialized view and do the whole query with the latest non-canonical chain tip, correct? I think this is something I'd need to fix in the current code.